server/models/DashBoard.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const User = require("./User");
+
+const TaskSchema = new Schema({
+  content: { type: String, required: true },
+  description: { type: String },
+  deadline: { type: String },
+  comments: [{ type: String }],
+  tag: { type: String },
+  action: { tag: { type: String }, color: { type: Number } }
+});
+
+const ColumnSchema = new Schema({
+  columnTitle: { type: String },
+  tasks: { type: Map, of: TaskSchema },
+  taskOrder: [{ type: mongoose.Schema.Types.ObjectId, ref: TaskSchema }] //don't populate this, it's for ordering
+});
+
+const DashBoardSchema = new Schema({
+  user: { type: mongoose.Schema.Types.ObjectId, ref: User },
+  dashboardTitle: { type: String, required: true },
```

Comment:
this is already in dashboard schema so you dont have to add dashboard to title. it could just be title. please change this for column schema as well and test to make sure everything still works properly and all the variables everywhere else have changed as well before you push.